Temperature_API/schemas.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class TemperatureBase(BaseModel):
+    date_time: datetime
+    temperature: float
+
+
+class TemperatureCreate(TemperatureBase):
+    pass
+
+
+class TemperatureList(TemperatureBase):
+    id: int
```

Comment:
The `TemperatureList` schema includes an `id` field, which is necessary for identifying individual temperature records. Ensure that this schema is used consistently across the API where a list of temperatures is returned.